scripts/run.sh

```diff
@@ -0,0 +1,2 @@
+# Run Docker container
+docker run -i vaccinewatcher:latest 
```

Comment:
What is the `-i` here for?

Reply:
I removed the -i per our conversation on slack. ✅